x/wasm/keeper/snapshotter.go

```diff
@@ -0,0 +1,153 @@
+package keeper
+
+// import (
```

Comment:
How can we solve this comment?
Please add as another issue the way to solve this.

Maybe, if we bump up to cosmos-sdk v0.45.5, is it solved?

Reply:
Yes, This should apply after bump up to cosmos-sdk v0.45.5